mautrix_telegram/__main__.py

```diff
@@ -149,6 +149,8 @@ async def resend_bridge_info(self) -> None:
     def prepare_stop(self) -> None:
         if self.periodic_sync_task:
             self.periodic_sync_task.cancel()
+        if self.as_connection_metric_task:
```

Comment:
This needs defining in the header too, otherwise Python will go NOOOO I CANT DEAL WITH `None`.

Reply:
Fixed in ac1a7283bbee691652ac4bc45158df42a2b0bc2a